api/controllers/itemsFetchController.js

```diff
@@ -0,0 +1,11 @@
+const itemsFetchService = require("../services/itemsFetchService");
+const { asyncErrorHandler } = require("../middleware/errorHandling");
+
+const itemsFetch = asyncErrorHandler(async (request, response) => {
```

Comment:
함수 이름을 동사형으로 잘 만들었네요. 
아래 예시는 일반적으로 쓰이는 예시이니 참고해 주세요.
동사를 가장 앞에쓰고(영문법이랑 비슷하게) 목적어를 사용하는 경우가 일반적이긴 합니다.
```suggestion
const getItemList = asyncErrorHandler(async (request, response) => {
```
or
```suggestion
const getItems = asyncErrorHandler(async (request, response) => {
```

Reply:
getItemsList로 변경했습니다!